packages/hdwallet-core/src/wallet.ts

```diff
@@ -203,6 +203,11 @@ export interface HDWallet extends HDWalletInfo {
    */
   getDeviceID (): Promise<string>
 
+  /**
+   * Retrieve the wallet's info
+   */
+  getDeviceInfo (): Promise<any>
```

Comment:
It would be better to put the new functionality here under the existing `getFeatures()` call, rather than adding new API surface.

Reply:
gotcha, I'll try that